packages/client/src/__tests__/integration/happy/enums/schema.prisma

```diff
@@ -3,30 +3,30 @@ datasource my_db {
   url      = "file:dev.db"
 }
 
+model Location {
+  city String @default("")
+  id   Int    @id @default(autoincrement())
+  User User[]
+}
+
+model Post {
+  createdAt DateTime @default(dbgenerated("'1970-01-01 00:00:00'"))
+  email     String   @unique(map: "Post.email") @default("")
```

Comment:
Is that map for "@unique(map: "Post.email")" really expected? 🤔

Reply:
https://github.com/prisma/prisma/pull/20075#discussion_r1254275417